integration_tests/commands/websocket/geo_tests.go

```diff
@@ -0,0 +1,87 @@
+package websocket
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Please shift to testify for assertions. You can refer to other test files for example.